docs/source/configuration.rst

```diff
@@ -33,6 +33,16 @@ task-based approach, when installing Calico with OpenStack on Ubuntu or
 Red Hat, please see :doc:`ubuntu-opens-install` or
 :doc:`redhat-opens-install`.
 
+System configuration
+--------------------
+
+A common problem on Linux systems is running out of space in the conntrack
+table, which can cause poor iptables performance. For this reason, we recommend
```

Comment:
I think this needs a bit more explanation how you can hit it (e.g. 'if you're running workloads that open many TCP/UDP connections').